configs/next/packages/gcc/stage_3

```diff
@@ -120,36 +147,86 @@ atcfg_configure() {
 			--without-libelf \
 			--with-as="${at_dest}/bin/as" \
 			--with-ld="${at_dest}/bin/ld" \
+			--with-stage1-ldflags="-Wl,--dynamic-linker=${ldso}" \
 			--with-cpu=${build_base_arch} \
 			--with-tune=${build_optimization}
 	else
-		echo "No configure on gcc_3 cross stage"
+		# Configure command for cross builds
+		CC="${system_cc}" \
+		CXX="${system_cxx}" \
+		LD_FOR_TARGET="${at_dest}/bin/${target}-ld" \
+		AS_FOR_TARGET="${at_dest}/bin/${target}-as" \
+		AR_FOR_TARGET="${at_dest}/bin/${target}-ar" \
+		NM_FOR_TARGET="${at_dest}/bin/${target}-nm" \
+		RANLIB_FOR_TARGET="${at_dest}/bin/${target}-ranlib" \
+		${ATSRC_PACKAGE_WORK}/configure \
+			--build=${host} \
+			--host=${host} \
+			--target=${target64:-${target}} \
+			--with-cpu=${default_cpu} \
+			--prefix=${at_dest} \
+			${with_longdouble:+--with-long-double-128} \
+			${secure_plt:+--enable-secureplt} \
+			${disable_multilib:+--disable-multilib} \
+			--with-advance-toolchain=$(basename ${at_dest}) \
+			--with-local-prefix=${at_dest} \
+			--enable-languages=c \
+			--enable-__cxa_atexit \
+			--enable-threads=posix \
+			--disable-libmudflap \
+			--disable-libssp \
+			--disable-libgomp \
+			--disable-libatomic \
+			--disable-libquadmath \
+			--disable-lto \
+			--disable-decimal-float \
+			--disable-libcc1 \
+			--disable-plugin \
+			--with-mpfr-include="${tmp_dir}/include" \
+			--with-mpfr-lib="${tmp_dir}/lib" \
+			--with-gmp-include="${tmp_dir}/include" \
+			--with-gmp-lib="${tmp_dir}/lib" \
+			--with-mpc-include="${tmp_dir}/include" \
+			--with-mpc-lib="${tmp_dir}/lib" \
+			--without-ppl \
+			--without-cloog \
+			--with-host-libstdcxx="-L${libdir} -L${tmp_dir}/lib -lstdc++ -lsupc++ -lm -lgmp -lgmpxx -lmpfr -lmpc" \
+			--with-build-sysroot="${dest_cross}" \
+			--with-sysroot="${dest_cross}" \
+			--with-native-system-header-dir="/usr/include"
 	fi
 }
 
 # Make build command
 atcfg_make() {
-	if [[ "${cross_build}" == "no" ]]; then
-		PATH=${at_dest}/bin:${PATH} \
-			${SUB_MAKE} all
-	else
-		echo "No make all on gcc_3 cross stage"
-	fi
+	PATH=${at_dest}/bin:${PATH} \
+		${SUB_MAKE} all
 }
 
 # Conditional install build command
 atcfg_install() {
-	if [[ "${cross_build}" == "no" ]]; then
-		make install -j1 DESTDIR=${install_place}
-	else
-		echo "No make install on gcc_3 cross stage"
-	fi
+	make install -j1 DESTDIR=${install_place}
 }
 
 # Conditional post install settings or commands to run
 atcfg_post_install() {
-	if [[ "${cross_build}" == "no" ]]; then
-		# Remove unnecessary python scripts
-		find "${install_place}/${at_dest}/lib" "${install_place}/${at_dest}/lib64" -name "libstdc++.so.*-gdb.py" -print -delete
```

Comment:
unrelated change? (also tied to the `find` command "added" well below)

Reply:
Likewise for the cross compiler.
We have to run this command both on native and cross builds.